<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrrigationSystemLiquidWidget</class>
 <widget class="QWidget" name="IrrigationSystemLiquidWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="btnStyle" stdset="0">
   <string>lblHintStyle</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>177</width>
     <height>368</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>40</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="lblHint">
      <property name="styleSheet">
       <string notr="true">font-size:24px;
font-weight:bold;</string>
      </property>
      <property name="text">
       <string>灌注系统液体！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>18</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnIrrigate">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>135</width>
          <height>135</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>135</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="btnStyle" stdset="0">
         <string>iconOnlyStyle</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnWash">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>系统液管路充灌</string>
        </property>
        <property name="btnStyle" stdset="0">
         <string>normalBtn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnFill">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>跳过充灌</string>
        </property>
        <property name="btnStyle" stdset="0">
         <string>normalBtn</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
